README.md

```diff
@@ -25,32 +26,26 @@
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+-Github flow knowledge.
+-Basic CSS and HTML.
+-Installation of files.
 
 ### Setup
+-To clone the repository run `[https://github.com/NATASHA-ct/MY-PORTFOLIO.git]` in your terminal
 
```

Comment:
- [ ] This link provided here is incorrect. I believe it should be: [https://github.com/NATASHA-ct/Leaderboard.git](https://github.com/NATASHA-ct/Leaderboard.git)

- [ ] Additionally, for the benefit of new users, after cloning the repository kindly include more specific instructions on how to run the app locally, like so:
  - run `cd Leaderboard`
  - run `npm install`
  - run `npm run build`
  - open `dist/index.html`